src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/Logging/RazorLogHubTraceProvider.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Composition;
+using System.Diagnostics;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.ServiceHub.Framework;
+using Microsoft.VisualStudio.LogHub;
+using VSShell = Microsoft.VisualStudio.Shell;
+
+namespace Microsoft.VisualStudio.LanguageServerClient.Razor.Logging
+{
+    [Shared]
+    [Export(typeof(RazorLogHubTraceProvider))]
+    internal class RazorLogHubTraceProvider
+    {
+        private static readonly LoggerOptions _logOptions = new LoggerOptions(
+            privacySetting: PrivacyFlags.CanContainPersonallyIdentifibleInformation | PrivacyFlags.CanContainPrivateInformation,
+            systemTags: new string[] { "Debug", "Build" });
```

Comment:
What do these system tags do?

Reply:
They essentially allow tagging the logs with the source of the log event. I had initially captured this from the wiki, but it appears we don't really need it for our purposes. Removing, thanks!